pkgs/development/libraries/caliper/default.nix

```
@@ -0,0 +1,96 @@
+{ lib, stdenv, fetchurl, python3, cmake, adiak, papi
+, withCuda ? false, cudatoolkit
+, withRocm ? false, rocm-runtime
+, withFortran? false, gfortran
+, withOpenmpi? false, openmpi
+, withMpich ? false, mpich
+, libunwindSupport ? true, libunwind
+, libpfmSupport ? true, libpfm
+, elfutilsSupport ? true, elfutils
```

Do we really need options for all of them?

```suggestion
, withCuda ? false, cudatoolkit
, withRocm ? false, rocm-runtime
, withFortran ? false, gfortran
, withOpenmpi ? false, openmpi
, withMpich ? false, mpich
, libunwindSupport ? true, libunwind
, libpfmSupport ? true, libpfm
, elfutilsSupport ? true, elfutils
```